src/main/java/io/github/opencubicchunks/cubicchunks/chunk/biome/CubeBiomeContainer.java

```diff
@@ -15,21 +15,21 @@
     private static final int SIZE_BITS = (int)Math.round(Math.log(16.0D) / Math.log(2.0D)) - 2;
     public static final int BIOMES_SIZE = 1 << SIZE_BITS + SIZE_BITS + SIZE_BITS;
 
-    private CubeBiomeContainer()
+    private CubeBiomeContainer(IObjectIntIterable<Biome> intIterable)
     {
-        this(new Biome[BIOMES_SIZE]);
+        this(intIterable, new Biome[BIOMES_SIZE]);
     }
 
-    public CubeBiomeContainer(Biome[] biomesIn) {
-        super(biomesIn);
+    public CubeBiomeContainer(IObjectIntIterable<Biome> indexedIterable, Biome[] biomesIn) {
+        super(indexedIterable, biomesIn);
     }
 
-    public CubeBiomeContainer(PacketBuffer packetBufferIn) {
-        super(packetBufferIn);
-    }
+//    public CubeBiomeContainer(PacketBuffer packetBufferIn) {
+//        super(packetBufferIn);
+//    }
```

Comment:
Why is this commented out? If it's not there in the vanilla equiv class just remove it.
Otherwise ask me

Reply:
otherwise I think it looks good, some annoying reformatting of stuff. If you're doing reformatting do all of it in a reformatting PR